templates/proposals/_entry.twig

```diff
@@ -0,0 +1,94 @@
+{% extends "_layout" %}
+
+{% block main %}
+
+    {% set image = entry.heroFooterImage.one() %}
+
+    <style>
+        .hero {
+            background-image: url("{{ image.url }}");
+        }
+    </style>
+
+    <section class="hero">
+        <div class="content-top">
+            <h2>{{ entry.clientName }}</h2>
+            <p>{{ entry.date|date('m/d/Y') }}</p>
+        </div>
+        <div class="content-main">
+            <h1>Super Duper Client - {{ entry.projectName }}</h1>
+        </div>
+        <div class="content-bottom">
+            <p>A PROPOSAL FROM ALRIGHT STUDIO</p>
+        </div>
+    </section>
+
+    <section id="body">
+        <div class="cover-letter">
+            <h2>Dear {{ entry.clientName }},</h2>
+            {% for block in entry.body %}
```

Comment:
Happy to look at this in person - could you explain your goal here? It appears you're looping through the `body` blocks multiple times throughout the page to target specific fields.

Reply:
Hey yeah sure no problem. So when I tried to use just one loop for all the elements, the hardcoded elements would repeat for every single block looped through. i.e., instead of 9 <hr> elements in DOM, it looped through 9 * 9 times, creating 81 <h4> elements. I wasn't sure how to resolve this, and this seemed to solve it.

Again, happy to explain this irl as well!